### PR TITLE
GH-127 moving the integration test with re-usable `testers` in a new `aio-lib-java-events-test` module

### PR DESCRIPTION
**Pull request by francoisledroff**
moving the integration test with re-usable `testers` in a new `aio-lib-java-events-test` module
see see https://github.com/adobe/aio-lib-java/issues/127

### CODE REVIEW

File: events_test/src/main/java/com/adobe/aio/event/management/ProviderServiceTester.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          eventMetadata.get().getDescription());
+      Assert.assertEquals(eventMetadataInput.getEventCode(), eventMetadata.get().getLabel());
+      logger.info("Added EventMetadata `{}` to AIO Events Provider `{}`", eventMetadata,
+          providerId);
+    }
+    return provider.get();
+  }
+
+
+  public void deleteProvider(String providerId) {
```
Comment (sangeetha5491) [inline events_test/src/main/java/com/adobe/aio/event/management/ProviderServiceTester.java:108]: many times i have seen that when an assertion fails, the test stops and the provider that is created is not cleaned up. Then we end up with a lot of test providers in that org. Can we ensure that the cleanup happens even if the test fails?

Comment (sangeetha5491) [reply events_test/src/main/java/com/adobe/aio/event/management/ProviderServiceTester.java:108]: same for other tests

Comment (francoisledroff) [reply events_test/src/main/java/com/adobe/aio/event/management/ProviderServiceTester.java:111]: thanks. done

**Review by sangeetha5491 — APPROVED** [review]
LGTM

**Pull request merged**